File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        mx=0
        curr=0
        for i in range (len(nums)):
            if(nums[i]==1):
                 curr+=1
            else:
                mx=max(mx,curr)
                curr=0
        mx=max(curr,mx)
        return mx 
        

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.cpp
class Solution {
public:
    vector<int> productExceptSelf(vector<int>& nums) {
        int n=nums.size();
        vector<int> v(n);
        for(int i=0;i<n;i++){
            v[i]=1;
            for(int j=0;j<n;j++){
                if(i!=j) v[i]*=nums[j];   
            }
        }
        return v;
    }
};
